### PR TITLE
Disable Scroll html

### PR DESCRIPTION
**Pull request by jstr14**
add disableScroll property to avoid scrolling views behind lightbox

### CODE REVIEW

**Comment by pexea12** [comment]
I have just commented on the changes of your pull request.

I would approve your pull request if you keep the style unchanged and change only what necessary.

File: src/components/script.js [hunk 1]
```diff
@@ -6,94 +6,98 @@ export default {
...
+
+    disableScroll: {
+      type: Boolean,
+      default: false
     },
 
     showLightBox: {
       type: Boolean,
-      default: true,
+      default: true
```
[the remark this answers is not in the document]
Comment (jstr14) [reply src/components/script.js]: Thanks for your reply I will change it

File: src/components/style.css [hunk 2]
```diff
@@ -256,3 +256,7 @@ img.vue-lb-modal-image {
 .fade-enter, .fade-leave-to {
   opacity: 0;
 }
+
+.no-scroll {
+  overflow-y: hidden;
+}
```
Comment (pexea12) [inline src/components/style.css]: A little change: can you add a new line to the end of this file? Sorry I didn't notice it until now.

**Comment by pexea12** [comment]
https://github.com/pexea12/vue-image-lightbox/releases/tag/5.7.0

**Pull request merged**